core/module.py

```diff
@@ -152,6 +152,53 @@ def describe(self) -> Union[Dict, None]:
             "handler": ".".join((handler, self.__class__.__name__.lower())),
         }
 
+        func = inspect.getmembers(self, predicate=inspect.ismethod)
+        filtered_funcs = []
+        for fn in func:
+            if not fn:
+                continue
+            f = fn[1]
+
+            viur_flags = getattr(f, "viur_flags", {})
+
+            attributes = {
+                "exposed": viur_flags.get("exposed", False),
+                "method": viur_flags.get("method", []),
+                "skey": "skey" in viur_flags
+            }
+
+            if attributes["exposed"]:
+                _params = {}
+                signature = inspect.signature(f)
+
+                annotations = get_type_hints(f)
+
+                for parameter in signature.parameters.values():
+                    _params[parameter.name] = {
+                        "type": "-",
+                        "default": None,
+                    }
+
+                    if parameter.annotation is not inspect.Parameter.empty:
+                        _params[parameter.name]["type"] = str(parameter.annotation)
+
+                    if parameter.default is not inspect.Parameter.empty:
+                        _params[parameter.name]["default"] = str(parameter.default)
+
+                return_hint = annotations.get("return")
+                attributes |= {
+                    "name": f.__name__,
+                    "docs": f.__doc__,
+                    "args": _params,
+                    "return": "-" if return_hint is None else str(return_hint)
```

Comment:
"-" could be interpreted as True. Shouldn't the client reading the data then choose output appropriate for the usecase